api/image.js

```diff
@@ -88,15 +90,15 @@ module.exports = async (request, response) => {
 		const queryParams = new URLSearchParams();
 		queryParams.set('theme', settings.theme);
 		queryParams.set('language', settings.language);
-		queryParams.set('lineNumbers', settings.lineNumbers);
+		queryParams.set('showLineNumbers', settings.showLineNumbers);
 		queryParams.set('code', settings.trimmedCodeSnippet);
 		queryParams.set('showBackground', settings.showBackground);
 		queryParams.set('hideButtons', settings.hideButtons);
 		if (settings.backgroundImage) { queryParams.set('backgroundImage', settings.backgroundImage); }
 		if (settings.backgroundColor) { queryParams.set('backgroundColor', settings.backgroundColor); }
 		if (settings.padding) { queryParams.set('padding', settings.padding); }
 
-		const pageUrl = `${hostname}/code.html?${queryParams.toString()}`;
+		const pageUrl = `${hostname}/private/preview?${queryParams.toString()}`;
```

Comment:
Why is this the page URL? shouldn't it just be `/preview`?

Reply:
I initially wanted to keep it secret so you cant fuck with the query params too much

Reply:
But it's still gonna be a public URL lol, it's how people are gonna use the API

Reply:
no they will be using `/api/image`, not the preview url which is basically just for taking a screenshot through pupeteer